spyder/config/main.py

```diff
@@ -307,6 +307,7 @@
               'console/use_project_or_home_directory': False,
               'console/use_cwd': True,
               'console/use_fixed_directory': False,
+              'startup/use_fixed_directory': False,

```

Comment:
Why is this is necessary? If it's only required for the new tests, this should be removed from here.

Reply:
It is required for correcting the bug

Reply:
Ok, thanks for the clarification @steff456!